data/json/furniture_and_terrain/furniture_vitrified.json

```
@@ -0,0 +1,880 @@
+    "max_volume": "2000 L",
+    "bash": {
+      "str_min": 12,
+      "str_max": 40,
+      "sound": "crunch!",
+      "sound_fail": "clink.",
+      "items": [ { "item": "black_glass_shard", "count": [ 40, 80 ] } ]
+    }
+  },
+  {
+    "type": "furniture",
+    "id": "f_vitrified_fridge",
+    "name": "black glass refrigerator",
+    "symbol": "{",
+    "description": "A black glass refrigerator, its door hanging open.  Inside is a seamless surface with protrusions resembling fresh fruit and plastic bottles.",
+    "color": "dark_gray",
+    "move_cost_mod": -1,
+    "coverage": 90,
+    "required_str": 10,
+    "flags": [ "BLOCKSDOOR", "MINEABLE" ],
+    "deconstruct": { "items": [ { "item": "black_glass_shard", "count": [ 40, 80 ] } ] },
+    "max_volume": "2000 L",
+    "bash": {
+      "str_min": 12,
+      "str_max": 40,
+      "sound": "crunch!",
+      "sound_fail": "clink.",
+      "items": [ { "item": "black_glass_shard", "count": [ 40, 80 ] } ]
+    }
+  },
+  {
+    "type": "furniture",
+    "id": "f_vitrified_armchair",
+    "name": "black glass armchair",
+    "symbol": "H",
+    "description": "An armchair that was presumably once very plush and comfortable.  Now the overstuffed cushions are hard, cold, and slippery.",
+    "color": "dark_gray",
+    "move_cost_mod": 1,
+    "coverage": 45,
+    "comfort": 1,
+    "floor_bedding_warmth": 0,
+    "required_str": 7,
+    "flags": [ "TRANSPARENT", "MOUNTABLE", "CAN_SIT", "ALIGN_WORKBENCH" ],
+    "bash": {
+      "str_min": 10,
+      "str_max": 40,
+      "sound": "crunch!",
+      "sound_fail": "clink.",
+      "items": [ { "item": "black_glass_shard", "count": [ 40, 80 ] } ]
+    }
+  },
+  {
+    "type": "furniture",
+    "id": "f_vitrified_chair",
+    "name": "black glass chair",
+    "symbol": "h",
+    "description": "A simple glass chair.  The seat is very thin, and crunches ominously under any weight.",
+    "color": "dark_gray",
+    "move_cost_mod": 1,
+    "coverage": 40,
+    "comfort": 0,
+    "floor_bedding_warmth": -1500,
+    "bonus_fire_warmth_feet": 1000,
+    "required_str": 4,
+    "flags": [ "TRANSPARENT", "MOUNTABLE", "CAN_SIT", "ALIGN_WORKBENCH" ],
+    "bash": {
+      "str_min": 6,
+      "str_max": 12,
+      "sound": "smash!",
+      "sound_fail": "whump.",
+      "items": [ { "item": "black_glass_shard", "count": [ 40, 80 ] } ]
+    }
+  },
+  {
+    "type": "furniture",
+    "id": "f_vitrified_desk",
+    "name": "black glass desk",
+    "symbol": "#",
+    "description": "A glass desk.  There are a handful of shapes atop it that suggest pens, paper, and miscellaneous knickknacks that are too jagged to identify.",
+    "color": "dark_gray",
+    "move_cost_mod": 1,
+    "coverage": 45,
+    "required_str": 5,
+    "crafting_pseudo_item": "medium_surface_pseudo",
+    "flags": [ "TRANSPARENT", "CONTAINER", "PLACE_ITEM", "MOUNTABLE", "FLAT_SURF", "SHARP", "PAINFUL" ],
```

```suggestion
    "flags": [ "TRANSPARENT", "PLACE_ITEM", "MOUNTABLE", "FLAT_SURF", "SHARP", "PAINFUL" ],
```
Can't contain anything if drawers don't open.

---

a throwback from when I was trying to randomize the key item locations into places that might include drawers. I never did figure out how to make one item only spawn once but with a chance of it being in a variety of spots.